types/conversations.d.ts

```diff
@@ -13,7 +79,14 @@ export interface StartConversationParameter {
   channelId?: string;
 }
 
-export type ChannelStatus = 'inactive' | 'active' | 'pending' | 'activation_required' | 'activation_code_required' | 'activating' | 'deleted';
+export type ChannelStatus =
+  | "inactive"
```

Comment:
Hey @mapacheverdugo,

Thanks for your MR, can you please revert changes on quotes from " to ' please?